Leaflet-finder/leaflet-dask-array.py

```diff
@@ -56,20 +45,20 @@ def Leaflet_finder(block, atoms, cutoff, len_chunks, block_id=None):
         start1 = time.time()
         atom = np.load(traj)
         len_chunks = int(len(atom)/np.sqrt(N_procs))
-        dist_matrix = da.zeros((len(atom),len(atom)), dtype=float, chunks=(len_chunks,len_chunks))        
+        dist_matrix = da.zeros((len(atom),len(atom)), dtype=float, chunks=(len_chunks,len_chunks))
 
         start2 = time.time()
 
         a = dist_matrix.map_blocks(Leaflet_finder, atom, 15.0, len_chunks, dtype=np.object).compute()
```

Comment:
It is used here `a = dist_matrix.map_blocks(Leaflet_finder, atom, 15.0, len_chunks, dtype=np.object).compute()`. dist_matrix is the whole dask array which we are working on its blocks using map_blocks.